public/sample-rubric.json

```diff
@@ -1,7 +1,7 @@
 {
     "id": "RBK1",
-    "formatVersion": "1.0.0",
-    "version": "12",
+    "version": "1.0.0",
+    "versionNumber": "12",
```

Comment:
`versionNumber` is meant to track the number of times this particular piece of rubric has been modified.  We aren't really using it, but it was part of the original discussion about rubric work. 

I think it could probably be removed.